client-apis/encrypt.md

```diff
@@ -67,24 +72,31 @@ A bound on the length of the [plaintext](#plaintext) to encrypt.
 This behavior MUST output the following if the behavior is successful:
 
 - [message](#message.md)
-
-This behavior MAY output the following:
-
 - [keyring trace](#structures.md#keyring-trace)
 
 To construct the outputs, some fields MUST be constructed using information obtained
 from a set of valid [encryption materials](#structures.md#encryption-materials).
 This behavior MUST obtain this set of [encryption materials](#structures.md#encryption-materials)
 by calling [Get Encryption Materials](#cmm-interface.md#get-encryption-materials) on a [CMM](#cmm-interface.md).
+
 The CMM used MUST be the input CMM, if supplied.
 If instead the user supplied a [keyring](#keyring-interface.md), this behavior MUST use a [default CMM](#default-cmm.md),
 constructed using the user supplied keyring as input.
-The call to [Get Encryption Materials](#cmm-interface.md#get-encryption-materials) MUST include the
-input [encryption context](#encryption-context), if supplied.
-If the length is known on the input [plaintext](#plaintext), this call MUST also include that value
-
-The [algorithm suite](#algorithm-suites.md) used in all aspects of this behavior MUST be the algorithm suite in the
+The call to [Get Encryption Materials](#cmm-interface.md#get-encryption-materials) on that CMM
+MUST be constructed as follows:
+
```

Comment:
This seems like a lot of whitespace?

Reply:
Newline before a list is standard throughout the spec. It shouldn't affect how it's rendered on github, but some things (like pandocs) end up requiring it to properly render the list.